### DISCUSSION TITLE
shift + key + mouseclick

### DISCUSSION THREAD

**Discussion by gralternate** [rustdesk/rustdesk#10391] [General]
### Bug Description

When holding (shift + any key) and then doing a mouse click, it does not register them together as (shift + any key + mouse click). 

I can confirm that holding shift then pressing a key will register as (shift + any key). 

### How to Reproduce

Try holding (shift + any key) and then doing a mouse click.

### Expected Behavior

Holding (shift + any key) and then mouse clicking should register together as (shift + any key + mouseclick)

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 10 -> linux rhel 9

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.6 -> 1.3.6

### Screenshots

I cannot post screenshots, I am sorry.

**Comment by rustdesk** [comment]
@fufesou  low priority

**Reply by rustdesk** [reply]
looks same to https://github.com/rustdesk/rustdesk/discussions/10385

**Reply by thenetguy** [reply]
@rustdesk no this is not a low priority , this severely affects my ability to game at work .please fix ASAP
